directory.py

```diff
@@ -42,11 +44,37 @@ def user_from_identity(sub) -> Dict[str, Any]:
         raise UserNotFoundError
 
 
+def insert_todo(todo: Todo):
+    try:
+        ds().set_object(
+            object_type="resource", object_id=todo.ID, display_name=todo.Title, properties={}
+        )
+
+        ds().set_relation(
+            subject_id=todo.OwnerID,
+            subject_type="user",
+            object_id=todo.ID,
+            object_type="resource",
+            relation="owner",
+        )
+    except NotFoundError:
+        raise UserNotFoundError
+
+
+def delete_todo(todoId: str):
+    try:
+        ds().delete_object(
+            object_type="resource", object_id=todoId, with_relations=True
+        )
+    except:
+        print("Error")
```

Comment:
I don't think that swallowing the exception and printing a meaningless message helps anyone. Why do we even need this try/except block?